cvat-core/src/server-proxy.ts

```diff
@@ -2150,8 +2179,44 @@ async function updateQualitySettings(
     }
 }
 
+async function getConsensusSettings(
+    filter: APIQualitySettingsFilter,
+): Promise<SerializedConsensusSettingsData> {
+    const { backendAPI } = config;
+
+    try {
+        const response = await Axios.get(`${backendAPI}/consensus/settings`, {
+            params: {
+                ...filter,
+            },
+        });
+
+        return response.data.results[0];
```

Comment:
Why we always take the first item from the list? 
Why not the last one?

Reply:
This is being used for `consensus settings` and `quality settings` for a given filter. We would get a list containing a single element.